src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
@dennypage SOP on pfSense is to rewrite at boot time to ensure consistency over all. Having to explain each write isn't something that matters here since it's done in numerous other places at boot time to ensure consistency.

@overhacked If you are going to verify, do a full checksum (sha256 or better). Don't just check size. No reason for half measures if you're going through the trouble.

@NOYB Please stop with the "nonsensical" line. It is completely reasonable to want to fully ensure a consistent state at all times. No matter what you believe, it is not "nonsense" to want to be absolutely certain you have good data. If they were rewritten at each filter sync your argument might hold water but it's at boot time or when the settings are saved. It's in line with how the rest of the system operates currently. That may change on the 3.x line but for now, it's what we want. Period.